### PR TITLE
Sunset /guide folder; Fix links 

### PR DESCRIPTION
**Pull request by angrycub**
Reparented some elements into docs.

### CODE REVIEW

**Comment by angrycub** [comment]
Linkchecker results:

```
➜ node_modules/dart-linkcheck/bin/linkcheck localhost:9999                                 
Perfect. Checked 141515 links, 998 destination URLs (586 ignored).
```

File: website/pages/api-docs/nodes.mdx [hunk 1]
```diff
@@ -825,15 +825,15 @@ $ curl \
 This endpoint toggles the drain mode of the node. When draining is enabled, no
 further allocations will be assigned to this node, and existing allocations will
 be migrated to new nodes. See the [Workload Migration
-Guide](/guides/operations/node-draining) for suggested usage.
+Guide](ttps://learn.hashicorp.com/nomad/operating-nomad/node-draining) for suggested usage.
```
Comment (jescalan) [inline website/pages/api-docs/nodes.mdx]: typo?

File: website/pages/docs/commands/node/drain.mdx [hunk 2]
```diff
@@ -134,4 +134,4 @@ $ nomad node drain -self -monitor
 [eligibility]: /docs/commands/node/eligibility
 [migrate]: /docs/job-specification/migrate
 [node status]: /docs/commands/node/status
-[workload migration guide]: /guides/operations/node-draining
+[workload migration guide]: ttps://learn.hashicorp.com/nomad/operating-nomad/node-draining
```
Comment (jescalan) [inline website/pages/docs/commands/node/drain.mdx]: another one

File: website/pages/docs/job-specification/migrate.mdx [hunk 3]
```diff
@@ -44,7 +44,7 @@ stanza for allocations on that node. The `migrate` stanza is for job authors to
 define how their services should be migrated, while the node drain deadline is
 for system operators to put hard limits on how long a drain may take.
 
-See the [Workload Migration Guide](/guides/operations/node-draining) for details
+See the [Workload Migration Guide](ttps://learn.hashicorp.com/nomad/operating-nomad/node-draining) for details
```
Comment (jescalan) [inline website/pages/docs/job-specification/migrate.mdx]: one more

File: website/pages/docs/upgrade/index.mdx [hunk 4]
```diff
@@ -126,7 +126,7 @@ Use the same actions in step #2 above to confirm cluster health.
 Following the successful upgrade of the servers you can now update your
 clients using a similar process as the servers. You may either upgrade clients
 in-place or start new nodes on the new version. See the [Workload Migration
-Guide](/guides/operations/node-draining) for instructions on how to migrate running
+Guide](ttps://learn.hashicorp.com/nomad/operating-nomad/node-draining) for instructions on how to migrate running
```
Comment (jescalan) [inline website/pages/docs/upgrade/index.mdx]: last one?

File: website/pages/docs/upgrade/upgrade-specific.mdx [hunk 5]
```diff
@@ -298,7 +298,7 @@ The `drain` command now blocks until the drain completes. To get the Nomad
 0.7.1 and earlier drain behavior use the command: `nomad node drain -enable -force -detach <node-id>`
 
 See the [`migrate` stanza documentation][migrate] and [Decommissioning Nodes
-guide](/guides/operations/node-draining) for details.
+guide](ttps://learn.hashicorp.com/nomad/operating-nomad/node-draining) for details.
```
Comment (jescalan) [inline website/pages/docs/upgrade/upgrade-specific.mdx]: i guess not

File: website/pages/api-docs/acl-policies.mdx [hunk 6]
```diff
@@ -20,8 +20,8 @@ to the region, and may lag behind the authoritative region.
 | `GET`  | `/acl/policies` | `application/json` |
 
 The table below shows this endpoint's support for
-[blocking queries](/api#blocking-queries), [consistency modes](/api#consistency-modes) and
-[required ACLs](/api#acls).
+[blocking queries](/api-docs#blocking-queries), [consistency modes](/api-docs#consistency-modes) and
```
Comment (schmichael) [inline website/pages/api-docs/acl-policies.mdx]: The `/api` -> `/api-docs` change makes me sad, but if our tools force that on us that's fine. What's another mountain of redirects? :grimacing:

**Pull request merged**